sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample15_RecognizeIdDocumentsFromFile.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        [Test]
+        public async Task RecognizeIdDocumentsFromFile()
```

Comment:
Since it's unique to ID documents, I think it may be worth illustrating the use of the `Gender` value type as well. I can understand if you think it's better to hold on this decision until we have a concrete reply from the service team regarding the field naming.

Reply:
yeah... I prefer to wait on that. We do provide the link with the list of fields and we do say that not all fields are here, so it should be ok